include/powsybl/iidm/Identifiable.hpp

```diff
@@ -42,7 +43,19 @@ class Identifiable : public virtual Validable, public Extendable, public MultiVa
 
     void addAlias(const std::string& alias);
 
-    const std::set<std::string>& getAliases() const;
+    void addAlias(const std::string& alias, bool ensureAliasUnicity);
+
+    void addAlias(const std::string& alias, const std::string& aliasType);
+
+    void addAlias(const std::string& alias, const char* aliasType);
```

Comment:
It is because of implicit cast of char* -> bool...
Calling `addAlias("alias", "test")` actually calls `addAlias(const std::string&, bool)`, which is really dangerous. This overload prevents this.